src/biokbase/narrative/system.py

```diff
@@ -37,31 +38,39 @@ def system_variable(var: str) -> Union[str, int, None]:
     var = var.lower()
     if var == "workspace":
         return os.environ.get("KB_WORKSPACE_ID")
-    elif var == "workspace_id":
+
+    if var == "workspace_id":
         ws_name = os.environ.get("KB_WORKSPACE_ID")
-        if ws_name is None:
-            return None
-        try:
-            ws_info = clients.get("workspace").get_workspace_info(
-                {"workspace": ws_name}
-            )
-            return ws_info[0]
-        except Exception:
-            return None
-    elif var == "user_id":
+        if ws_name:
+            try:
+                ws_info = clients.get("workspace").get_workspace_info(
+                    {"workspace": ws_name}
+                )
+                return ws_info[0]
+            # TODO: catch specific exception(s)
+            except Exception:
+                # do nothing
+                pass
```

Comment:
unless the workspace name can be retrieved from the workspace info, this fn returns none, so no need to add another `return None` to the `except` statement